specs/lib/moveTechniqueRenderStatesSpec.js

```diff
@@ -0,0 +1,219 @@
+'use strict';
+var Cesium = require('cesium');
+var moveTechniqueRenderStates = require('../../lib/moveTechniqueRenderStates');
+
+var WebGLConstants = Cesium.WebGLConstants;
+
+var gltf = {
+    programs: {
+        program_0: {
+            attributes: [
+                'a_normal',
+                'a_position',
+                'a_texcoord0'
+            ],
+            fragmentShader: 'BoxTextured0FS',
+            vertexShader: 'BoxTextured0VS'
+        }
+    },
+    shaders: {
+        BoxTextured0FS: {
+            type: WebGLConstants.FRAGMENT_SHADER,
+            uri: 'BoxTextured0FS.glsl'
+        },
+        BoxTextured0VS: {
+            type: WebGLConstants.VERTEX_SHADER,
+            uri: 'BoxTextured0VS.glsl'
+        }
+    },
+    techniques: {
+        technique0: {
+            states: {
+                enable: [
+                    WebGLConstants.DEPTH_TEST,
+                    WebGLConstants.BLEND,
+                    WebGLConstants.CULL_FACE
+                ]
+            }
+        }
+    },
+    materials: [
+        {
+            technique: 'technique0'
+        }
+    ]
+};
+
+describe('updateMaterialProperties', function() {
+    it('sets material.doubleSided property if CULL_FACE is not enabled', function () {
+        var gltfWithUpdatedMaterials = moveTechniqueRenderStates(gltf);
```

Comment:
`gltf` should be cloned first.

Reply:
This may also be causing intermittent failures when tests are run in different order, which I think jasmine does now?